core/tests/protractor_utils/users.js

```diff
@@ -85,6 +85,11 @@ var logout = async function() {
 // that this will fail if the user already has a username.
 var _completeSignup = async function(username) {
   await waitFor.pageToFullyLoad();
+  let cookieBanner = element(
+    by.css('.protractor-test-oppia-cookie-banner-accept-button'));
+  if (await cookieBanner.isPresent()) {
+    await action.click('Cookie banner', cookieBanner);
```

Comment:
Can we change the element name (in `action.click()`) to 'Cookie Banner Accept button'?

Reply:
done